### DISCUSSION TITLE
Notebook ou script?

### DISCUSSION THREAD

**Discussion by Marine1719** [BIO6032/2022-SEIR-reproduction#5] [General]
Est-ce qu'on part vers un notebook ou un script pour la programation? On peut peu-être fonctionner par vote? Je propose de liker l'un ou l'autre des commentaires pour voir les préférences. 

- [ ] Notebook

**Comment by Marine1719** [comment]
- [x]  Script

**Reply by blanchette31** [reply]
Personnellement, je préfère utiliser un script + literate.jl plutôt qu'un Notebook.
